test/ITensorNetworks/models.jl

```diff
@@ -0,0 +1,13 @@
+using ITensors
+using ITensorNetworkAD
+using ITensorNetworkAD.ITensorNetworks: Models
+
+@testset "test local hamiltonian builder" begin
+  Nx = 2
+  Ny = 3
+  sites = siteinds("S=1/2", Nx * Ny)
```

Comment:
A heads up that I made a definition of `siteinds` that accepts multiple coordinates and returns a Matrix of Index. See: https://github.com/ITensor/ITensorNetworkAD.jl/blob/main/src/ITensorNetworks/tensor_networks.jl#L20-L25

Reply:
Feel free to turn it into a future issue to make use of that functionality, it is not the most important thing right now but I wanted to make you aware.

Reply:
Sounds good, just fixed this